lib/markdown-preview-view.coffee

```diff
@@ -177,6 +183,28 @@ class MarkdownPreviewView extends ScrollView
   getGrammar: ->
     @editor?.getGrammar()
 
+  getDocumentStyleSheets: -> # This function exists so we can stub it
```

Comment:
One problem is that we won't always be loading the syntax theme style sheets into the global document context. We do that now only to support disabling the shadow DOM, which is something we're going to disable as soon as we have time. A better approach would be to get the styles out of an existing `atom-text-editor` element *or* create and attach an `<atom-styles context="atom-text-editor">` element and get the styles out of that. Sorry for the confusion. We really need to drop the global styles sooner rather than later.